DataFormats/L1TCorrelator/src/classes_def.xml

```diff
@@ -0,0 +1,137 @@
+
+<lcgdict>
+
+ <class name="l1t::TkPrimaryVertex" ClassVersion="14">
+  <version ClassVersion="14" checksum="2843779965"/>
+  <version ClassVersion="13" checksum="304492084"/>
+  <version ClassVersion="12" checksum="515857374"/>
```

Comment:
Here and below: the convention for new classes (that have never been stored) is to start versioning with 3, and without any development-time intermediate versions.

Reply:
done.